config/site.php

```diff
@@ -156,6 +151,10 @@
                         ],
                         'title' => __('Videos'),
                     ],
+                    'security' => [
+                        'url' => 'https://book.cakephp.org/4.0/en/contributing/tickets.html#reporting-security-issues',
```

Comment:
This link would be more permanent https://github.com/cakephp/cakephp#security

Reply:
Done